skywalking/plugins/sw_bottle.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from skywalking import Layer, Component, config
+from skywalking.trace.carrier import Carrier
+from skywalking.trace.context import get_context, NoopContext
+from skywalking.trace.span import NoopSpan
+from skywalking.trace.tags import TagHttpMethod, TagHttpURL
+
+link_vector = ['http://bottlepy.org/docs/dev/']
+support_matrix = {
+    'bottle': {
+        '>=0.12': ['*']
```

Comment:
This isn't a supported syntax, and you confused the left and right side. 

left side should be version of Python right side is the package, see and example below.
```
support_matrix = {
    'hug': {
        '>=3.10': ['2.5', '2.6'],  # api deprecated for 3.10
        '>=3.6': ['2.4.1', '2.5', '2.6'],  # support begins 2.4.1
    }
}
```